apps/src/templates/instructions/ReviewTab.jsx

```diff
@@ -1,22 +1,54 @@
 import React, {Component} from 'react';
-//import PropTypes from 'prop-types';
+import {getStore} from '@cdo/apps/redux';
 import javalabMsg from '@cdo/javalab/locale';
 import Comment from './codeReview/Comment';
-import {demoComments} from './codeReview/commentShape';
 import CommentEditor from './codeReview/CommentEditor';
+import * as codeReviewDataApi from './codeReview/codeReviewDataApi';
 
 export default class ReviewTab extends Component {
-  // Once we have real comments
-  // static propTypes = {
-  //   comments: PropTypes.arrayOf(commentShape)
-  // };
-
   state = {
-    readyForReview: false
+    readyForReview: false,
+    comments: [],
+    token: '',
+    forceRecreateEditorKey: 0
+  };
+
+  componentDidMount() {
+    const channelId = getStore().getState().pageConstants.channelId;
+
+    // projectVersion = 'latest' is a placeholder until we implement
+    // storing project version when saving comments
+    codeReviewDataApi
+      .getCodeReviewCommentsForProject(channelId, 'latest')
```

Comment:
could using 'latest' potentially cause issues when we do store project version with comments (ex we are still querying for latest but now all comments have versions so we don't get any results)?

Reply:
I was assuming we had to finish the project version stuff before the pilot, I meant short term placeholder

Reply:
I think our plan for pilot is to show comments for all versions, is that right @jmkulwik?

Reply:
Ooh interesting, my understanding was we show all comments on all versions of a given project, just style comments on older versions differently. In any case, if this isn't a feature we 100% need by the pilot, I could make project version nullable (it currently is not, as I thought it was one of the primary identifiers for a comment)

Reply:
yep your understanding is correct, but we won't be doing that pre-pilot. Why do we want to query comments by version since we always want them all anyway?

Reply:
🤦 totally don't need to query using it :) I'll remove the code where we're querying with it, and modify the AJAX calls to only include the channelId (and just set up the controller to put in a temporary placeholder value). I'll write a migration to make the field nullable, and remove the temporary placeholder version string being used in the controller at that point.